### PR TITLE
Fix `MD053/link-image-reference-definitions`

### PR DESCRIPTION
**Pull request by xtqqczze**
[Link and image reference definitions should be needed](https://github.com/DavidAnson/markdownlint/blob/main/doc/md053.md)

### CODE REVIEW

**Comment by mairaw** [comment]
Do you wanna resolve the merge conflicts here @xtqqczze?

File: release-notes/5.0/5.0-supported-os.md [hunk 1]
```diff
@@ -45,32 +44,20 @@ Note: .NET 5 requires OpenSSL 1.x. Newer distro versions such as Ubuntu 22.04 us
 [Alpine-lifecycle]: https://wiki.alpinelinux.org/wiki/Alpine_Linux:Releases
 [CentOS]: https://www.centos.org/
 [CentOS-lifecycle]:https://wiki.centos.org/FAQ/General
```
Comment (mairaw) [inline release-notes/5.0/5.0-supported-os.md:46]: it seems this is the archived version?
```suggestion
[CentOS-lifecycle]:https://wiki.centos.org/FAQ(2f)General.html
```

Comment (xtqqczze) [reply release-notes/5.0/5.0-supported-os.md:46]: Yes, see https://github.com/dotnet/core/pull/9594

**Comment by xtqqczze** [comment]
@mairaw merge conflicts are resolved

**Review by richlander — APPROVED** [review]

**Comment by richlander** [comment]
Thanks!

**Pull request merged**